evalml/pipelines/utils.py

```diff
@@ -261,6 +283,7 @@ def _get_preprocessing_components(
         list[Transformer]: A list of applicable preprocessing components to use with the estimator.
     """
     if is_time_series(problem_type):
+        # --> add to ts and test for ts
```

Comment:
What do you mean by this?  Did you intend to add the ordinal encoder here?

Reply:
My plan was to add the ordinal encoder wherever the one hot encoder was being used, and I don't see a reason why it couldn't be used for time series, so I was just leaving a note to myself to add `_get_ordinal_encoder` below as well and test it.